src/pages/About.tsx

```diff
@@ -52,6 +53,10 @@ const AboutPage = () => {
   const styles = useAboutStyles();
   const { data } = useServerMetadataQuery();
   const { t } = useTranslation();
+
+  const currentPaths = useMemo(() => [], []);
```

Comment:
why is `useMemo` used here when it can be a constant?

Reply:
have to be memoised otherwise the breadcrumb will trigger infinity rerendering.

Reply:
Even on constants?

Reply:
It is possible to be constant if is defined outside the scope of the component.